scripts/create-tf-backend.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
```

Comment:
Because we're using ephemeral accounts in PR envs, we need to recreate our TF backend whenever we get a clean account.